src/posthog-core.ts

```diff
@@ -14,6 +14,7 @@ import {
 import { DeadClicksAutocapture, isDeadClicksEnabledForAutocapture } from './extensions/dead-clicks-autocapture'
 import { ExceptionObserver } from './extensions/exception-autocapture'
 import { errorToProperties } from './extensions/exception-autocapture/error-conversion'
+import { HistoryAutocapture } from './extensions/history-autocapture'
```

Comment:
Most of your PRs auto-format the imports, which adds noise to the PR review.
Can you double-check that your auto-format config is the same as everyone else's?

Reply:
it's caused by a setting I have on VS Code. I [created a PR](https://github.com/PostHog/posthog-js/pull/1914) to prevent it from having effect in this repo